odoo/tests/common.py

```diff
@@ -2040,6 +2040,17 @@ def _get_modifier(self, field, modifier, default=False, modmap=None, vals=None):
                         # field value should be [(6, _, ids)], we want just the ids
                         field_val = field_val[0][2] if field_val else []
 
+                    # in an embedded view, the modifiers' view is not self._view so the fields for the domains cannot be
+                    # found but we have the field name and it's value, so we can guess a m2m field in a hack-ish way
+                    if f_['type'] is None and f.endswith('_ids') and len(field_val) == 1:
+                        try:
+                            if field_val[0][0] == 6:
```

Comment:
use command constants instead of hardcoding 6: `Command.SET`

Reply:
There are lots of places in the file that use hard-coded numeric commands, but I fixed this one only.